frontend/src/routes/_student/(dashboard)/-components/progress-overview/progress-overview.tsx

```diff
@@ -0,0 +1,39 @@
+"use client"
+
+import { Progress } from "@/components/ui/progress"
+import { $api } from "@/lib/api"
+import { FC } from "react"
+
+const STUDENT_ID = 1 // TODO: change this to the actual student ID
```

Comment:
This is because the authentication and user management APIs are not yet implemented. Once these features are added, the hardcoded values will be removed and replaced with actual user IDs.